pubsub/hazelcast/hazelcast.go

```diff
@@ -154,7 +154,7 @@ func (l *hazelcastMessageListener) handleMessageObject(message []byte) error {
 	return pubsub.RetryNotifyRecover(func() error {
 		l.p.logger.Debug("Processing Hazelcast message")
 
-		return l.pubsubHandler(&pubsubMsg)
+		return l.pubsubHandler(context.Background(), &pubsubMsg)
```

Comment:
I think here we could use `l.p.ctx` instead of `context.Background()` so that cancelation works if the component is closed.

Reply:
Done 👍